### PR TITLE
chore: update Item component test

### PR DESCRIPTION
**Pull request by jrr997**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |update Item component test case|
| 🇨🇳 Chinese |更新Item组件测试|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by jrr997** [comment]
background: https://github.com/ant-design/ant-design/pull/34653

**Comment by zombieJ** [comment]
https://github.com/react-component/picker/pull/362  @afc163 pls help to check it's affected by this?

**Comment by afc163** [comment]
Please rebase master.

File: components/locale-provider/__tests__/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -249003,25 +249003,25 @@ exports[`Locale Provider should display the text as pt 1`] = `
                     <thead>
                       <tr>
                         <th>
-                          Seg
+                          2ª
```
Comment (afc163) [inline components/locale-provider/__tests__/__snapshots__/index.test.js.snap:249006]: Don't change this.

**Pull request closed without merging**